docs/supported-tools.rst

```diff
@@ -0,0 +1,260 @@
+Supported Tools and Dependencies
+================================
+
+Read the Docs supports two tools to build your documentation:
+`Sphinx <https://www.sphinx-doc.org/>`__ and `MkDocs <https://www.mkdocs.org/>`__.
+In order to provide :doc:`several features </features>`,
+Read the Docs needs to inject or modify some content while building your docs.
+
+When an incompatible change happens in one of these tools or dependencies,
+we need to change our code as well to keep our features working.
+This is done with backwards compatibility in mind,
+but sometimes is hard to keep compatibility with old and new versions at the same time.
+In order to continue moving forward on future development and features,
+we need to drop support for some versions.
+
+.. note::
+
+   Your existent documentation will always be kept online and working,
+   but **when support for a tool or dependency ends, new builds may fail**.
+
+.. contents:: Contents
+   :local:
+   :depth: 3
+
+End of support policy
+---------------------
+
+Our policy defines how long a given tool or dependency is considered supported.
+Read the Docs will contact all users when an end of support date is close,
+after that date your builds may start failing and you will need to upgrade in order to receive support.
```

Comment:
@humitos wouldn't you say that this is possible?

```suggestion
Read the Docs will share advice when a end date for support is known.
If detection of tool usage is possible, users will be contacted directly.
After the end date, your builds may start failing and you will need to upgrade or change the tool in order to receive support.
```

Reply:
Yeah, we are tracking "all packages installed in the virtualenv" now. So, if we want, we could contact "all users building their docs with `Sphinx==3.2.1`"